sys/build.sh

```diff
@@ -38,7 +38,7 @@ BuildJobsThrottler(){
 	echo "So, the build will run on $MAKE_JOBS job(s)."
 }
 
-if [ "`uname`" = Linux ]; then
+if [ "$(uname)" = Linux ]; then
```

Comment:
i saw there are plenty of `uname` cmds. can you convert it to a local variable and use that in `if` and `elif` ?